code/modules/shuttles/shuttle_supply.dm

```diff
@@ -77,5 +77,8 @@
 
 //returns the ETA in minutes
 /datum/shuttle/autodock/ferry/supply/proc/eta_minutes()
-	var/ticksleft = arrive_time - world.time
-	return round(ticksleft/600,1)
+	return round((arrive_time - world.time) / (1 MINUTE)) // Floor, so it's an actual timer
```

Comment:
Won't this make all the PDAs, computers, etc which call this proc report "0" minutes remaining when there are still 59 seconds until arrival?